Tests/Microsoft.AppCenter.Analytics.Test.Windows/SessionTrackerTest.cs

```diff
@@ -97,22 +129,33 @@ public void ResumeAfterShortPause()
         public void HandleEnqueuingLogDuringSession()
         {
             _sessionTracker.Resume();
-            var eventLog = new EventLog { Timestamp = DateTime.Now };
+            var eventLog = new EventLog();
             _mockChannelGroup.Raise(group => group.EnqueuingLog += null, null, new EnqueuingLogEventArgs(eventLog));
             Assert.IsNotNull(eventLog.Sid);
             Assert.AreNotEqual(Guid.Empty, eventLog.Sid);
         }
 
+        /// <summary>
+        ///     Verify that an enqueuing log with already specified timestamp is ignored from session tracker, like crash logs.
+        /// </summary>
+        [TestMethod]
+        public void HandleEnqueuingLogWithExplicitTimestamp()
+        {
+            _sessionTracker.Resume();
+            var eventLog = new EventLog { Timestamp = DateTime.Now };
```

Comment:
I could see the point of the test but it might mislead. Can't you use anonymous type of log or custom type of log here?
Not a big deal though.

Reply:
We should use the real log type here, we send only events from Analytics.